README.md

```diff
@@ -1,10 +1,11 @@
 # KCron Common
 Cron realization for Kotlin Multiplatform
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Scogun_kcron-common&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Scogun_kcron-common) ![GitHub](https://img.shields.io/github/license/Scogun/kcron-common?color=blue) ![Publish workflow](https://github.com/Scogun/kcron-common/actions/workflows/publish.yml/badge.svg) [![Maven Central with version prefix filter](https://img.shields.io/maven-central/v/com.ucasoft.kcron/kcron-common/0.5.3?color=blue)](https://search.maven.org/artifact/com.ucasoft.kcron/kcron-common/0.5.3/jar)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Scogun_kcron-common&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Scogun_kcron-common) ![GitHub](https://img.shields.io/github/license/Scogun/kcron-common?color=blue) ![Publish workflow](https://github.com/Scogun/kcron-common/actions/workflows/publish.yml/badge.svg) [![Maven Central with version prefix filter](https://img.shields.io/maven-central/v/com.ucasoft.kcron/kcron-common/0.5.4?color=blue)](https://search.maven.org/artifact/com.ucasoft.kcron/kcron-common/0.5.3/jar)
 
 ### Features
-* Kotlin Multiplatform library
+* Kotlin Multiplatform library (with support for jvm, Linux, Windows 
```

Comment:
Let's write support targets like a sublist:
````
Support:
  * JVM
  * Linux
  * ...
```

Reply:
Done in 0099a2bc5cec40bed599ed0ce1414a93c7a27118